File: Services/src/ProEventos.API/Model/Evento.cs
namespace ProEventos.API.Model
{
    public class Evento
    {
        public int EventoId { get; set; }
        public string Local { get; set; }
        public string DataEvento { get; set; }
        public string Tema { get; set; }
        public string QudPessoas { get; set; }
        public string Lote { get; set; }
        public string ImagemURL { get; set; }
    }
}

File: SPA/ProEventos-App/src/app/models/Evento.ts
import { Lote } from "./Lote";
import { Palestrante } from "./Palestrante";
import { RedeSocial } from "./RedeSocial";

export interface Evento {
   id: number;
   local: string;
   dataEvento?: Date;
   tema: string;
   qudPessoas: number;
   imagemURL: string;
   telefone: string;
   email: string;
   lotes: Lote[];
   redesSociais: RedeSocial[];
   palestrantesEventos: Palestrante[];
}